.travis.yml

```diff
@@ -18,19 +18,23 @@ install:
   - export PATH="$HOME/miniconda/bin:$PATH"
   - hash -r
   - conda config --set always_yes yes --set changeps1 no
-  - conda update -q conda
+  - conda config --add channels conda-forge
+  - conda config --add channels skywalker-tag
+  - conda config --add channels lightsource2-tag
+  - conda install conda-build
+  - conda update -q conda conda-build
   # Useful for debugging any issues with conda
   - conda info -a
   # Grab all dependencies
   # Build dependencies
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION setuptools -c defaults -c conda-forge
+  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION setuptools
   - source activate test-environment
   # Test suite dependencies
-  - conda install coverage pip wheel pytest flake8 codecov -c defaults -c conda-forge
-  # Module test dependencies
-  - conda install ipython pyyaml coloredlogs ophyd pcds-devices -c defaults -c conda-forge -c lightsource2-tag -c skywalker-tag
-  # Build it!
-  - python setup.py install
+  - conda install -q coverage pip wheel pytest flake8 codecov
+  # Test conda build
+  - conda build -q conda-recipe
+  # Install it
+  - conda install -q hutch-python --use-local
```

Comment:
So this installs using the conda-recipe specs? I like this... Rather have an up to date conda-recipe than `requirements.txt`